.gitignore

```diff
@@ -13,6 +13,7 @@ xxh32sum
 xxh64sum
 xxh128sum
 xxhsum
+!programs/xxhsum
```

Comment:
Do you plan on having multiple program targets in `programs/`  ?

Reply:
Hmm
I don't plan on it.

What about another path idea, `bin`?

Reply:
`bin/` is usually employed to host the generated artifacts, not the sources.

Coming back to `programs/xxhsum/`,
I believe we already had this discussion, and there were many candidates to choose from.
I'm fine with most of them, be it `xxhsum-src/`, `cli/`, `frontend/`, or even `programs/` (directly).
`xxhsum/` would have been great, though it's apparently a problem for `Windows`.

Reply:
Well I am not sure now. Tell me what you want, it is your decision.

Also, `xxhsum/` would be a problem for non-Windows, as if one ran `git pull` before `make clean`, `git` will try to overwrite the *executable* `xxhsum` with the *folder* `xxhsum/`.

```
$ make
...
cc xxhsum.o xxhash.o xxhsum
$ file xxhsum
xxhsum: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, not stripped
$ git pull
...
error: The following untracked working tree files would be overwritten by merge:
        xxhsum
        xxhsum
        xxhsum
        xxhsum
        xxhsum
        xxhsum
Please move or remove them before you merge.
Aborting

```

Reply:
OK, let's go for `cli` then